packages/c/Cuckoo.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Cuckoo",
+  "description": "A blog theme based on Typecho",
+  "keywords": [
+    "cuckoo",
+    "blog"
+  ],
+  "license": "GPL-3.0",
+  "homepage": "https://github.com/bhaoo/Cuckoo",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/bhaoo/Cuckoo.git"
+  },
+  "authors": [
+    {
+      "name": "Bhao"
+    }
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/bhaoo/Cuckoo.git",
+    "fileMap": [
+      {
+        "basePath": "assets",
+        "files": [
+          "**/*"
```

Comment:
We want to avoid using open globstars when we can as this may mean we match files that cdnjs doesn't support, and leaves cdnjs very vulnerable to incredibly large versions being matched accidently.

```suggestion
          "css/*.css",
          "js/*.js",
          "fonts/**/*.@(woff|woff2)",
          "icons/**/*.@(woff|woff2)",
          "images/**/*.@(png|jpg|jpeg|gif)"
```